CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+4.68.0
+==========
+### Features
+- Added WithFmiPath() api to support FMI scenarios in MSAL. See [Issue #5110](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/5110)
+- Added WithAdditionalCacheKeyComponents() to enable the modification access token of cache keys with the provided components. See [Issue #5091](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/5091)
+
+### Bug Fixes
+- Reordered the condition for ManagedIdentitySource.MachineLearning to be checked after ManagedIdentitySource.AppService instead of before it. See [Issue #5077]([https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/commit/e22ca8a2d8b37232f6878782746e76d0f9d9ad50](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/5077))
```

Comment:
```suggestion
- Reordered the condition for ManagedIdentitySource.MachineLearning to be checked after ManagedIdentitySource.AppService instead of before it. See [Issue #5077](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/5077)
```